server/src/configs/app.js

```diff
@@ -12,7 +12,7 @@ const app = (
   const create = (
 
   ) => {
-    const portToUse = process.env.APP_PORT;
+    const portToUse = process.env.PORT;
```

Comment:
import all the env from a constant file

Reply:
Pushed #74 to solve this. Please take a look.